### PR TITLE
Update old dbtoken-service documentation

### PR DESCRIPTION
**Pull request by bennerv**
### Which issue this PR addresses:

Fixes old dbtoken-service docs that aren't relevant.

### CODE REVIEW

**Review by mbarnes — APPROVED** [review]
Thanks for this!  LGTM

File: docs/dbtoken-service.md [hunk 1]
```diff
@@ -47,10 +47,26 @@ in depth in the face of an attack on the gateway component.
...
+
+   ```bash
+   AZURE_DBTOKEN_CLIENT_ID="$(az ad app create --display-name dbtoken \
+      --oauth2-allow-implicit-flow false \
+      --query appId \
+      -o tsv)"
+
+   OBJ_ID="$(az ad app show --id $AZURE_DBTOKEN_CLIENT_ID --query id)"
+
+   > __NOTE:__: the graph API requires this to be done from a managed machine
```
Comment (SudoBrendan) [inline docs/dbtoken-service.md:60]: super-nit, but this doesn't format correctly in github, should we just make it a `#` instead of `>__NOTE:__`?

**Review by SudoBrendan — APPROVED** [review]
one nit, lgtm

File: docs/dbtoken-service.md (hunk 1 above)
Comment (cadenmarchese) [reply docs/dbtoken-service.md:60]: I'm seeing that too - it looks like the note is meant to be outside of the markdown and instead it's getting included in the markdown.

Comment (bennerv) [reply docs/dbtoken-service.md]: Done - good catch.  Thank you :)

**Pull request merged**